docs/docs/tutorial/chapter2/getting-dynamic.md

```diff
@@ -57,6 +85,8 @@ For the tutorial we're keeping things simple and using an integer for our ID col
 
 Integers make for nicer URLs like https://redwoodblog.com/posts/123 instead of https://redwoodblog.com/posts/eebb026c-b661-42fe-93bf-f1a373421a13.
 
+The MongoDB is using strings as `id` which is mapping to the mongo built-in object id(`_id`)
```

Comment:
Tried to improve the copy a bit here. Is this still accurate?
```suggestion
The MongoDB example uses `String` since it maps to MongoDB's built-in object id (`_id`)
```